docs/emr.rst

```diff
@@ -0,0 +1,33 @@
+.. _presto-admin-on-emr-label:
+..
+.. If you modify this file, you will have to modify the NOTEs in the following files:
+.. docs/installation/java-installation.rst
+.. docs/installation/presto-admin-configuration.rst
+.. docs/installation/presto-admin-installation.rst
+..
+
+===============================================
+Setting up Presto Admin on Amazon EMR cluster 
+===============================================
+
+To install, configure and run Presto Admin on Amazon EMR cluster, follow the instructions in :ref:`quick-start-guide-label`, but keep an eye on the notes or sections specfic to EMR cluster. We reiterate these EMR specific caveats below:
+
+- To install Presto Admin on Amazon EMR cluster, follow the instructions in :ref:`presto-admin-installation-label` except for the following difference: 
+
+	- Use online installer instead of offline installer (see :ref:`advanced-installation-options-label`).
+
+- To configure Presto Admin on Amazon EMR cluster, follow the instructions in :ref:`presto-admin-configuration-label`. Specifically, we recommend the following property values during the configuration: 
```

Comment:
The reason I didn't prefix Amazon EMR cluster with 'an' was that I was referring to the "Amazon EMR cluster" as a cluster category rather than as a singular Amazon cluster. Added it.